datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -202,14 +204,91 @@ export const REDSHIFT_PASSWORD: RecipeField = {
     rules: null,
 };
 
+export const TABLEAU_CONNECTION_URI: RecipeField = {
+    name: 'connect_uri',
+    label: 'Connection URI',
+    tooltip: 'Tableau host URL.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.connect_uri',
+    rules: null,
+};
+
+const tableauProjectFieldPath = 'source.config.projects';
+export const TABLEAU_PROJECT: RecipeField = {
```

Comment:
can we make the button for this field say "Add project" instead of "Add pattern" - but still say "Add pattern" for the other list fields?

Reply:
Checked and resolved.